hack/kcp-edge-bootstrap.sh

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 --create-folder --verbose
+
+# This script installs KCP-Edge binaries to a folder of choice
+#
+# Arguments:
```

Comment:
(1) This listing omits `--imw` and `--wmw`.

(2) I think that for this script, the command line would read more naturally if it looked like `--create-imw $name` and `--create-wmw $name`.